ClientSecurity/Configuring_IPS_Notifications.md

```diff
@@ -0,0 +1,61 @@
+# Configuring Intrusion Prevention System (IPS) Notifications
+
+## Overview
```

Comment:
why the switch to ## vs ### in this one vs the others?  all KB's generally use ###